source/cgame/blocklist.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#define	MAX_BLOCKS	1024
```

Comment:
```suggestion
#define	MAX_BLOCKLIST_ENTRIES	1024
```
Conveys the meaning a bit better.